theano/tensor/signal/tests/test_conv.py

```diff
@@ -82,13 +82,14 @@ def sym_conv2d(input, filters):
 
     def test_basic(self):
         """
-        Basic functionality of nnet.conv.ConvOp is already tested by
-        its own test suite.  We just have to test whether or not
-        signal.conv.conv2d can support inputs and filters of type
-        matrix or tensor3.
+        Basic functionality of nnet.abstract_conv.AbstractConvOp is
+        already tested by its own test suite.  We just have to test
+        whether or not signal.conv.conv2d can support inputs and
+        filters of type matrix or tensor3.
         """
         if(not theano.tensor.nnet.conv.imported_scipy_signal and
-           theano.config.cxx == ""):
+           theano.config.cxx == "" or
+           theano.config.mode == "FAST_COMPILE"):
             raise SkipTest("conv2d tests need SciPy or a c++ compiler")
```

Comment:
I think that condition should be rewritten.
AbstractConv should not depend on `scipy.signal` any more.